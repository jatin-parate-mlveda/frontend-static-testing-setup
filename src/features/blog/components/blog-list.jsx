export const BlogList = () => (
  <ul>
    <li>Blog 1</li>
    <li>Blog 2</li>
    <li>Blog 3</li>
    <li>Blog 4</li>
  </ul>
);
