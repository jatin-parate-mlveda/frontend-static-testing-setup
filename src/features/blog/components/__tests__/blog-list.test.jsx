import { render } from '@testing-library/react';
import { BlogList } from '../blog-list';

test('matches snapshot', () => {
  const { container } = render(<BlogList />);

  expect(container.outerHTML).toMatchSnapshot();
});
