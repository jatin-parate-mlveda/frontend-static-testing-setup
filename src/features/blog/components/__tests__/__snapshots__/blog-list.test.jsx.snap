// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `"<div><ul><li>Blog 1</li><li>Blog 2</li><li>Blog 3</li><li>Blog 4</li></ul></div>"`;
